src/backends/supported.jl

```diff
@@ -234,10 +234,11 @@ supportedArgs(::GRPackage) = [
   ]
 supportedAxes(::GRPackage) = _allAxes
 supportedTypes(::GRPackage) = [:none, :line, :path, :steppre, :steppost, :sticks,
-                                   :scatter, :heatmap, :hexbin, :hist, :density, :bar,
-                                   :hline, :vline, :contour, :path3d, :scatter3d, :surface, :wireframe]
+                               :scatter, :heatmap, :hexbin, :hist, :density, :bar,
+                               :hline, :vline, :contour, :path3d, :scatter3d, :surface,
+                               :wireframe, :ohlc, :pie]
```

Comment:
I thought you only support `[:none, :line, :scatter]` so far.  Why do you have so many listed?